src/sensor_motion.py

```diff
@@ -0,0 +1,23 @@
+import RPi.GPIO as GPIO
+from time import sleep
+
+GPIO.setmode(GPIO.BCM)
+GPIO.setwarnings(False)
+pir = 16
+GPIO.setup(pir, GPIO.IN)
+
+print('Initializing Sensor...')
+sleep(2)
```

Comment:
Is `sleep` necessary?

Reply:
Yes

Reply:
Why? If you remove the `sleep` is going to cause an error?

Reply:
It takes time for the sensor to initialize so we need to include the sleep to give the hardware time, then run the code.